src/PrinterLog.php

```diff
@@ -182,13 +182,17 @@ public function showMetrics(Printer $printer)
 
             foreach ($metrics as $key => $value) {
                 $label = $this->getLabelFor($key);
-                if ($label && $value > 0) {
+                if ($label && array_sum($metrics) > 0) {
                     $series[$key]['name'] = $label;
                     $series[$key]['data'][] = $value;
                 }
             }
         }
-
+        foreach ($series as $key => $value) {
+            if (array_sum($value['data']) == 0) {
```

Comment:
I suggest a small comment to help future readers as the printer metrics code can be a bit hard to understand overall, so any extra details are good to take

```suggestion
            // If the metric has a value of 0 for all dates, remove it from the data set
            if (array_sum($value['data']) == 0) {
```